code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -145,6 +145,13 @@
 	icon_state = "warden_envirohelm"
 	inhand_icon_state = "warden_envirohelm"
 
+/obj/item/clothing/head/helmet/space/plasmaman/security/head_of_security
+	name = "head of security's plasma envirosuit helmet"
+	desc = "A plasmaman containment helmet designed for the Head of Security, a pair of gold stripes being added to differeciate them from other members of security."
```

Comment:
here's the part where the english got so bad I just decided to suggest redoing them entirely. THEN, I realized this is already misspelled in the other plasmamen stuff that have it, dear god, save us all

```suggestion
	desc = "A special containment helmet designed for the Head of Security. A pair of gold stripes are added to differentiate them from other members of security."
```